metrics/README.md

```diff
@@ -134,6 +134,7 @@ The preferred API to store results is through the provided JSON API.
 The API provides two groups of functions:
```

Comment:
No longer two groups. Am I right?

Reply:
Indeed, you are correct. Reworded to try to make count independent :-)